apiclient/invoke.go

```diff
@@ -39,12 +39,12 @@ func (c *Client) Invoke(ctx context.Context, agentID string, input string, opts
 		close(events)
 		return &types.InvokeResponse{
 			Events:   events,
-			ThreadID: resp.Header.Get("X-Otto-Thread-Id"),
+			ThreadID: resp.Header.Get("X-Obot-Thread-Id"),
```

Comment:
Might make migration easier if we fall back to the old header if the new one isn't set.